site/blog/drafts/why-i-hate-markdown.rst

```diff
@@ -110,9 +110,15 @@ you can write:
 
 	.. warning:: This is a Warning!
 
+Or, in AsciiDoc, you can write:
```

Comment:
This example doesn't add anything :) Would be good to ahve an Adoc example somewhere else, so that it's more mixed between rst & adoc, not just "here's how to do the same thing in both" :)